services/blocking_recommendation_service.py

```
@@ -0,0 +1,276 @@
+            total_count=total_count,
+            percentage=round(percentage, 1),
+            daily_breakdown=daily_breakdown,
+            trend_direction=trend_direction,
+            trend_percentage_change=round(trend_pct_change, 1),
+            comparable_categories=comparable,
+            recommendation_factors=factors
+        )
+
+    def get_blocked_categories_for_account(
+        self,
+        email_address: str
```

Ruff (0.14.6)

Unused method argument: `email_address`

(ARG002)